yaml2ics.py

```diff
@@ -22,6 +22,8 @@
 def event_ics_from_yaml(event_yaml: dict) -> str:
     d = event_yaml
     repeat = d.pop('repeat', None)
+    if 'name' in d:
```

Comment:
We can probably skip this, since no one is currently using the "old" file format.

Reply:
Tests used it (thus the quick way to make tests not fail), but now tests are changed, too.  (would be nice to use the simpler `name` or `title`, but I guess diverging from raw ICS isn't worth it.